### ISSUE TITLE
File won't close on ctrl q -- vcv rack 'not responding'

### ISSUE THREAD

**Issue by snakeofjune** [VCVRack/Rack#1210]
See https://github.com/VCVRack/Rack/issues/1021 for issue report standards and https://github.com/VCVRack/Rack/issues/730 before posting any code or terminal output.

Operating system(s):
Version of Rack if using official binary, or commit hash and branch if compiling from source:
All hardware relevant to your issue (e.g. graphic card model, audio/MIDI device):

First of all, thanks so much for developing this. It's a really exciting project and is helping me to understand modular synthesis. I don't think there's this issue been reported before.

Rack has been working fine until recently: I've tried to control q the file to start on another file and I get the indefinite colour wheel. I've force quit Rack; reopened it a number of times and same issue. So at the moment it's somewhat unusable.

I'm using rack ver. 0.6.2c
MacOS Mojave: 10.14.3
Macbook pro mid 2015
2.6 intel course i7

Hope that's enough info.

Thanks!

**Comment by joemcmahon** [comment]
I can confirm this behavior. It's not consistent for me, but I do end up force-quitting Rack a fair amount. I'll capture the output and post it next time.

**Comment by AndrewBelt** [comment]
Does this happen when only VCV plugins are installed?

**Comment by joemcmahon** [comment]
Yes, I'm not using anything that doesn't get installed by Rack itself. I suspect there's something happening with the audio interfaces (I have Loopback installed) that's causing it to get all upset. As soon as I manage to hang it again I will get the crashlog.

**Comment by joemcmahon** [comment]
Okay, managed to get it to hang today, but I think this may be a different bug:
 - patch running
 - put the machine to sleep
 - reopen machine, no audio
 - attempt to select output in Audio module
 - Rack hangs

Force-quit did not give me a traceback. I will try building from source and see if I can duplicate this with the debugger running in Xcode.

**Comment by TheBrigadier** [comment]
Not sure if this is the right place or forum but suffering the same issues as snakeofjune
Rack v0.6.2.c
macOS 10.13.6
When try to open .vcv file or quit applicatin (cmd+q) spinning beach ball of death.
Worked until early May 2019 now cannot quit without force quitting and will not open any .vcv files created in this installed version from earlier. Cannot Clear patch and start over, hangs and have to force quit. Cannot open earlier work, hangs and have to force quit. Can add modules but only to the rack that opens. Are there any log files I should or could be looking at to see if the is a corrupted plugin for instance?
